src/environment_setup/biopam.mli

```diff
@@ -8,23 +8,22 @@ val default_opam_url : string
 (** The default location from where we download biopam. *)
 val default_biopam_url : string
 
-(** A workflow to make sure that Biopam is configured.*)
-val configured : ?biopam_home:string -> ?host:KEDSL.Host.t ->
-  install_path:string -> unit ->
-  < is_done : KEDSL.Condition.t option > KEDSL.workflow_node
-
+type tool_type = [
+  | `Library of string   (** The export variable that points to witness. *)
+  | `Application
+]
 (** The type of tool that we are installing via opam.
 
    This guides installation and determines:
     1. where we look for the {{!recfield:witness}witness} in
        [opam_install_path]/[package] or [opam_install_path]/bin.
     2. Whether we export $PATH (Application) or $LIBVAR (Library).*)
-type tool_type =
-  | Library of string   (** The export variable that points to witness. *)
-  | Application
 
 (** A description of what we'd like Biopam to install.*)
-type install_target = {
+type install_target = private {
```

Comment:
`private` in an `.mli` means that users can explore/match/destruct the values but cannot create new values. The goal is to force the use of `val install_target` below (general good practice to hide implementations + hold invariants in the module).